Library/Source/NativeEngine.cpp

```diff
@@ -768,7 +767,22 @@ namespace babylon
                 {
                     const uint32_t location = compiler.get_decoration(stageInput.id, spv::DecorationLocation);
                     AppendBytes(vertexBytes, bgfx::attribToId(static_cast<bgfx::Attrib::Enum>(location)));
-                    attributeLocations[stageInput.name] = location;
+                    
+                    std::string attributeName = stageInput.name;
```

Comment:
is this string replacement necessary for metal?

Reply:
Yes! same as opengl. Metal shader is text (not binary like d3d) so attributes have to match between bgfx and Babylon.

Reply:
Maybe we can add something to bgfx to allow it to use different names.

Reply:
Hardcoded names in bgfx https://github.com/bkaradzic/bgfx/blob/c6fc1b069d8f88646c702987ad1594c6e5d033ee/src/renderer_gl.cpp#L44
We can ask Branimir if id would accept a PR to allow for name change